Sources/Testing/Expectations/ExpectationChecking+Macro.swift

```diff
@@ -1142,15 +1142,15 @@ public func __checkClosureCall<R>(
 @_spi(Experimental)
 public func __checkClosureCall(
   exitsWith expectedExitCondition: ExitCondition,
-  performing body: @convention(thin) () async throws -> Void,
+  performing body: @convention(thin) () -> Void,
```

Comment:
This makes the type checker a bit happier as it gets motion sick dealing with `@convention(thin)`.

Reply:
Does this restrict the body closure of an exit test to not be async or throws?

Reply:
No, just the synthesized wrapper around the emitted type. This closure isn't actually ever called.